src/targets/gpu/target.cpp

```diff
@@ -132,14 +133,20 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         dead_code_elimination{},
         normalize_ops{},
         dead_code_elimination{},
+        // 1. For int4 ops: first remove the identity op next to packed int4 data.
```

Comment:
I dont really like having just in4 specific comments here as these passes cover more than just int4 cases, so I think it is confusing.

Reply:
Reverted.

Reply:
You are right. I have missed pushing this particular file! Let us wait what the perf results are anyways!